File: python/26/2/1/PrintedBook.py
from Book import Book
class PrintedBook(Book):
    def __init__(self, title, author, ISBN, price, stock,pageCount):
        super().__init__(title, author, ISBN, price, stock)
        self.__pageCount=pageCount
    def displayDetails(self):
        super().displayDetails()
        print("PageCount: ",self.__pageCount)
    

File: java/codelearn/PrintedBook.java
public class PrintedBook extends Book {
    private int pageCount;

    public PrintedBook(String title, String author, String ISBN, double price, int stock, int pageCount) {
        super(title, author, ISBN, price, stock);
        this.pageCount = pageCount;
    }

    @Override
    public void displayDetails() {
        super.displayDetails();
        System.out.println("PageCount: " + pageCount);
    }
}
